code/modules/clothing/spacesuits/plasmamen.dm

```diff
@@ -305,6 +305,11 @@
 	inhand_icon_state = null
 	armor_type = /datum/armor/plasmaman_engineering
 
+/datum/armor/plasmaman_engineering
```

Comment:
subtype any added plasmamen armor to the job they correspond to and edit the values as appropriate please